specs/RestartRequested.md

```diff
@@ -20,11 +20,20 @@ void WebView_CoreWebView2InitializationCompleted(object sender, CoreWebView2Init
     }
 }
 
-void WebView_RestartRequested(CoreWebView2Environment sender, object e)
+void WebView_RestartRequested(CoreWebView2Environment sender, CoreWebView2RestartRequestedEventArgs e)
 {
-    // Depending on your app experience, you may or may not
-    // want to prompt user to restart the app.
-    RestartIfSelectedByUser();
+    if (e.Priority == RestartRequestedPriority.Normal) 
+    {
+        // Depending on your app experience, you should remind the user
+        // to restart on normal cadence.
+        RemindToRestartForUpdate();
+    }
+    else
```

Comment:
Don't assume "!=Normal" means "High".  This encourages code for which if we defined a "Low" Priority enum value later, would treat High and Low the same.  (presumably the developer would want Low to behave like "Normal" in this sample case.)

Reply:
This is discussed in the above comment, thx

Reply:
Updated